### ISSUE TITLE
chain head command errors

### ISSUE THREAD

**Issue by ZenGround0** [filecoin-project/go-filecoin#2392]
**Describe the bug**
My chain is synced to the current head as verified by output of `chain ls` but `chain head` produces error: `Error: cid was incorrectly formatted`.  This does not appear to be sporadic, it happens for each head every time I try it.

**To Reproduce**
Sync your chain to the release branch test cluster and run `go-filecoin chain head` using a release branch build.

**Expected behavior**
chain head should be displayed instead of error

**Screenshots**

**Desktop (please complete the following information):**
 - OS: OSX
 - Version [e.g. 22]: b87348b354d8ffea5309cb6e395fdc82d0a9eef1 (release branch)

**Additional context**

**Comment by ZenGround0** [comment]
cc @phritz

**Comment by frrist** [comment]
Our encoder is expecting a slice of CID's: https://github.com/filecoin-project/go-filecoin/blob/master/commands/chain.go#L36
But we are emitting a slice of blocks: https://github.com/filecoin-project/go-filecoin/blob/master/commands/chain.go#L34

**Comment by frrist** [comment]
Something funky seems to be happening with the encoding:
```
frrist@pine ~/s/g/f/go-filecoin (master)> go-filecoin chain head --enc=json
[{"/":"zDPWYqFCrMH36tQoVUiLrAaC4GCwK4KefHL9JZXxGkcDZq3gTKLt"}]
```
```
13:57:15.612 DEBUG  cmds/http: incoming API request:  /chain/head?encoding=json&stream-channels=true handler.go:90
```
We seem to ignore the encoding flag on commands:
```
frrist@pine ~/s/g/f/go-filecoin (master)> go-filecoin chain head --enc=text
[{"/":"zDPWYqFCrMH36tQoVUiLrAaC4GCwK4KefHL9JZXxGkcDZq3gTKLt"}]
```
```
13:57:25.078 DEBUG  cmds/http: incoming API request:  /chain/head?encoding=json&stream-channels=true handler.go:90
```

**Comment by frrist** [comment]
closed via #2396
